build/commands/chat/user/ban.js

```diff
@@ -142,7 +145,7 @@ class UserBan extends _command.Command {
 
 UserBan.flags = {
   type: _command.flags.string({
-    char: 'u',
+    char: 't',
```

Comment:
@AnatolyRugalev this should reflect in docs too

Reply:
Forgot to regenerate the docs. Will fix it in next PR